.github/workflows/main.yaml

```diff
@@ -63,6 +63,16 @@ jobs:
           set -x
           targets=$(cat ${GITHUB_WORKSPACE}/.github/config/${{ inputs.platform }}.json | jq -cr '.targets  | join(" ")')
           echo "targets=${targets}" >> $GITHUB_OUTPUT
+      - id: set-build-configs
+        shell: bash
+        run: |
+          set -x
+          # Attempt to load build configs from JSON. If not present use default.
+          build_configs=$(cat ${GITHUB_WORKSPACE}/.github/config/${{ inputs.platform }}.json | jq -cr '.build_configs')
+          if [[ "${build_configs}" == "null" ]]; then
```

Comment:
I'm a little on the fence on just copying the defaults into every config for clarity, but i this works too.

Reply:
I opted for a default value to signify that setting this in the platform config is exceptional. We should not be overriding this lightly.